tests/helpers.py

```diff
@@ -527,3 +551,42 @@ def run_io_with_rados_bench(**kw):
     logger.info(ret)
     logger.info("Finished radosbench")
     return ret
+
+
+def get_all_pvs():
+    """
+    Gets all pv in given namespace
+
+    Returns:
+         dict: Dict of all pvc in namespaces
+    """
+    ocp_pv_obj = ocp.OCP(
+        kind=constants.PV, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    return ocp_pv_obj.get()
+
+
+@retry(AssertionError, tries=10, delay=5, backoff=1)
+def validate_pv_delete(pv):
+    """
+    validates if pv is deleted after pvc deletion
+
+    Args:
+        pv (str): pv from pvc to validates
+    Returns:
+        bool: True if deletion is successful
+
+    Raises:
+        AssertionError: If pv is not deleted
+    """
+    ocp_pv_list = get_all_pvs()
+    pv_all_list = ocp_pv_list['items']
+    if pv_all_list:
+        for item in pv_all_list:
+            logging.info(item['metadata']['name'])
+            logging.info(pv)
+            if item['metadata']['name'] == pv:
+                raise AssertionError
+            else:
+                return True
```

Comment:
Do we actually want to return True here? We will end up returning True for the first `item` that doesn't raise an `AssertionError` and fail to check the subsequent items in `pv_all_list`. Seems to me that we should remove this `else: return True` and only return True after all items in `pv_all_list` have been iterated over and none have raised an `AssertionError`.